appservice/src/tree/DeploymentTreeItem.ts

```diff
@@ -30,15 +30,16 @@ enum DeployStatus {
 }
 
 export class DeploymentTreeItem extends AzureTreeItem<ISiteTreeRoot> {
-    public static contextValue: string = 'deployment';
+    public static contextValue: string;
```

Comment:
This change breaks `showTreeItemPicker` (aka code [like this](https://github.com/Microsoft/vscode-azureappservice/blob/dc917610ae6968a6d2864a4e0b3ce4c81f904066/src/commands/deployments/redeployDeployment.ts#L11)). The static value can't be undefined

The simplest fix would be to define a static `contextValues` array with all the possible values, but IMO the better fix would be to modify `showTreeItemPicker` to work with a RegExp instead of a string (kind of like what you did for TestUserInput [here](https://github.com/Microsoft/vscode-azuretools/pull/389)).